docs/reference/content/guides/querying.md

```diff
@@ -0,0 +1,258 @@
+  saved. For example, in the example above if `found` is saved back to MongoDB, the `last_name` field that currently exists in the database 
+  for this entity will be removed.
+{{% /note %}}
+
+### Limiting and Skipping
+
+Pagination of query results is often done as a combination of skips and limits.  Morphia offers `Query.limit(int)` and `Query.offset(int)` 
+for these cases.  An example of these methods in action would look like this:
+
+```java
+datastore.createQuery(Person.class)
+    .offset(1)
+    .limit(10)
+    .asList()
+```
+
+This query will skip the first element and take up to the next 10 items found by the query.  There's a caveat to using skip/limit for 
+pagination, however.  See the [skip]({{< docsref "reference/method/cursor.skip" >}}) documentation for more detail.
+
+### Ordering
+
+Ordering the results of a query is done via [`Query.order(String)`](/javadoc/org/mongodb/morphia/query/Query.html#order-java.lang.String-)
+.  The javadoc has complete examples but this String consists of a list of comma delimited fields to order by.  To reverse the sort order
+ for a particular field simply prefix that field with a `-`.  For example, to sort by age (youngest to oldest) and then income (highest 
+ to lowest), you would use this:
+ 
+```java
+query.order("age,-income");
+```
+
+### Tailable Cursors
+
+If you have a [capped collection]({{< docsref "core/capped-collections/" >}}) it's possible to "tail" a query so that when new documents 
+are added to the collection that match your query, they'll be returned by the [tailable cursor]({{< docsref 
+"reference/glossary/#term-tailable-cursor" >}}).  An example of this feature in action can be found in the [unit tests]({{< srcref 
+"morphia/src/test/java/org/mongodb/morphia/TestQuery.java">}}) in the `testTailableCursors()` test:
+
+```java
+getMorphia().map(CappedPic.class);
+getDs().ensureCaps();                                                          // #1
+final Query<CappedPic> query = getDs().createQuery(CappedPic.class);
+final List<CappedPic> found = new ArrayList<CappedPic>();
+
+final Iterator<CappedPic> tail = query.tail();
+while(found.size() < 10) {
+    found.add(tail.next());                                                    // #2
+}
+```
+There are two things to note about this code sample:
+
+1.  This tells Morphia to make sure that any entity [configured](/guides/annotations/#entity) to use a capped collection has its collection 
+created correctly.  If the collection already exists and is not capped, you will have to manually [update]({{< docsref 
+"core/capped-collections/#convert-a-collection-to-capped" >}}) your collection to be a capped collection.
+1.  Since this `Iterator` is backed by a tailable cursor, `next()` will block until a new item is found.  If your application can't block
+ on `next()`, `hasNext()` works as you'd expect an Iterator to.  In this version of the unit test, we tail the cursor and pull out 
```

Comment:
What does "`hasNext()` works as you'd expect an Iterator to" mean in this context?  Does it always return true and not block?  Or does it also block?  Remember we dealt with this in the Java driver a while back...